recipe/meta.yaml

```diff
@@ -1,40 +1,52 @@
-{% set version = "2.3.9" %}
+{% set name = "spyder" %}
+{% set version = "3.1.2" %}
+{% set sha256 = "89a98164e0d27ede273c2b50c8c05044add66c55ab733634d972e1cc55448371" %}
 
 package:
-  name: spyder
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn:  spyder-{{ version }}.zip
-  url: https://pypi.io/packages/source/s/spyder/spyder-{{ version }}.zip
-  md5: dd01e07a77123c128ff79ba57b97c1d7
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 2
-  osx_is_app: True
+    entry_points:
+        - spyder = spyder.app.start:main
+    number: '0'
+    osx_is_app: 'True'
 
 requirements:
   build:
     - python
+    - sphinx
   run:
     - python
     - pyqt 4.11.*
-    - rope
+    - qtpy >=1.1
+    - qtawesome >=0.4.1
+    - rope >=0.9.4
```

Comment:
This needs to be as I told you

    - rope =0.9.4  # [py3k]
    - rope >=0.9.4  # [py2k]

in case people adds a rope feedstock later, because right now `0.9.4` is the only version compatible with Python 3 (and that's the one called `rope_py3k` in PyPi but repackaged simply as `rope` by Continuum :-)